OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -150,7 +150,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 				var minefield = GetMinefieldCells(minefieldStart, cell, Info.MinefieldDepth)
 					.Where(c => IsCellAcceptable(self, c) && self.Owner.Shroud.IsExplored(c)
-						&& movement.CanEnterCell(c, null, BlockedByActor.Immovable) && (mobile != null && mobile.CanStayInCell(c)))
+						&& movement.CanEnterCell(c, null, BlockedByActor.Immovable) && (mobile?.CanStayInCell(c) ?? false)
```

Comment:
There is a missing brace now:
```suggestion
						&& movement.CanEnterCell(c, null, BlockedByActor.Immovable) && (mobile?.CanStayInCell(c) ?? false))
```